libcore/src/Simulation.h

```diff
@@ -198,13 +185,9 @@ class Simulation
     bool correctGeometry(std::shared_ptr<Building> building, std::shared_ptr<TrainTimeTable>);
 
     /**
-     * Updates the output filename if the current file exceeds 10MB.
+     * Updates the output filename if the current file exceeds 16MB.
```

Comment:
I would prefer mentioning `_maxFileSize` here, instead of 16 MB.

Reply:
Fixed